src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,74 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo,
+  closeModal: () => void,
+};
+
+const emptyUser = {
+  id: 0,
+  name: '',
+  email: '',
+  phone: '',
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, closeModal }) => {
+  const [user, setUser] = useState<User>(emptyUser);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo.userId)
+      .then(currentUser => setUser(currentUser))
+      .finally(() => setIsLoading(false));
+  }, [todo.userId]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
+      {isLoading
+        ? <Loader />
+        : (
+          <div className="modal-card">
+            <header className="modal-card-head">
+              <div
+                className="modal-card-title has-text-weight-medium"
+                data-cy="modal-header"
+              >
+                {`Todo #${todo.id}`}
+              </div>
+              <button
+                type="button"
+                aria-label="close"
+                className="delete"
+                data-cy="modal-close"
+                onClick={closeModal}
+              />
+            </header>
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+            <div className="modal-card-body">
+              <p className="block" data-cy="modal-title">
+                {todo.title}
+              </p>
+
+              <p className="block" data-cy="modal-user">
+                {todo.completed
+                  ? <strong className="has-text-success">Done</strong>
+                  : <strong className="has-text-danger">Planned</strong>}
```

Comment:
```suggestion
                {todo.completed
                  ? <strong className="has-text-success">Done</strong>
                  : <strong className="has-text-danger">Planned</strong>
                }
```

Reply:
@Anastasiia-Svintsova I had problem with that. 
<img width="712" alt="image" src="https://user-images.githubusercontent.com/102159736/204095663-7c599f5d-0441-43c0-a09f-c4d16fcc2c2a.png">